metapool/sample_sheet.py

```diff
@@ -1820,21 +1836,24 @@ def make_sample_sheet(metadata, table, sequencer, lanes, strict=None):
         If a value is omitted from this dictionary the values in square
         brackets are used as defaults.
 
-        - Bioinformatics: List of dictionaries describing each project's
-          attributes: Sample_Project, QiitaID, BarcodesAreRC, ForwardAdapter,
-          ReverseAdapter, HumanFiltering, library_construction_protocol,
-          experiment_design_description
+        - Bioinformatics: List of dictionaries, one per project, describing
+          each project's attributes: Sample_Project, QiitaID, BarcodesAreRC,
+          ForwardAdapter, ReverseAdapter, HumanFiltering,
+          library_construction_protocol, experiment_design_description - Note
+          that the requirements will depend on the SheetType.
         - Contact: List of dictionaries describing the e-mails to send to
           external stakeholders: Sample_Project, Email
-
+        - SheetType: str, sample sheet type
+        - SheetVersion: str, the version of the sheet
+        - Assay: assay type for the sequencing run. No default value will be
+          set, this is required.
+        - SampleContext: list, can be empty
```

Comment:
As long as we are cleaning the docs up, please make this:

SampleContext: List of dictionaries, one per blank, containing Sample_Name, PrimaryQiitaStudy,, SecondaryQiitaStudies, and Sample_Type.  If empty, blanks are inferred from data by sample name.